plugins/zoom/Dockerfile

```diff
@@ -1,24 +1,18 @@
 FROM rapid7/insightconnect-python-3-38-plugin:4
```

Comment:
That's interesting. Does it really have to be the version `4` of the SDK? Or should it be `5` 🤔 
@mrinehart-r7

Reply:
attempting to run the task in postman after updating SDK to version 5 returns the following error:
`{
        "error": "not enough values to unpack (expected 3, got 2)",
        "log": "rapid7/Zoom:4.0.2. Step name: monitor_sign_in_out_activity\nFirst run\nGetting events from 2023-06-26T15:15:28Z until 2023-06-27T15:15:28Z\nCalling GET https://api.zoom.us/v2/report/activities\nGot response status code: 401\nReceived HTTP 401, re-authenticating to Zoom...\nCalling Zoom API to refresh OAuth token...\nGot status code 200 from OAuth token refresh\nRequest for new OAuth token was successful!\nCalling GET https://api.zoom.us/v2/report/activities\nGot response status code: 200\nGot 4 events from Zoom!\nLatest event time is: 2023-06-27T14:26:52Z\nUpdated state, state is now: {'boundary_events': ['9f683d4d635a549ddf152d0b810bbd82a9ce33ff'], 'last_request_timestamp': '2023-06-27T15:15:28Z', 'latest_event_timestamp': '2023-06-27T14:26:52Z', 'status_code': 200}\nnot enough values to unpack (expected 3, got 2)\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-5.1.4-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 424, in handle_step\n    output, state, has_more_pages = self.start_step(\n  File \"/usr/local/lib/python3.8/site-packages/insightconnect_plugin_runtime-5.1.4-py3.8.egg/insightconnect_plugin_runtime/plugin.py\", line 582, in start_step\n    output, state, has_more_pages = func(params, state)\nValueError: not enough values to unpack (expected 3, got 2)\n",
        "meta": {},
        "status": "error"
    }`

Reply:
Do not update this to version 4 yet - the code isn't ready to support the `has_more_pages` return output yet.